### PR TITLE
Updated tutorial for 0.11.1

### PR DESCRIPTION
**Pull request by justsayno**
I have updated all the screenshots and version numbers and ran through this tutorial on a fresh install on Windows to be sure.

As I said in an email to Eddie I did use Cmder and windows its so the screen shots are a little different in look now. I hope this is not a problem. I have updated everything to be pretty consistent though and should be fine.

Also like I said I'll need some help to get the yeoman site running on my computer. I did all the steps on the readme, no errors but Grunt serve just gives me a blank page (nothing at all). Not sure why...

Let me know what you think I am happy to keep revising until its good.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -2,6 +2,7 @@ GEM
   remote: http://rubygems.org/
   specs:
     RedCloth (4.2.9)
+    RedCloth (4.2.9-x86-mingw32)
```
Comment (eddiemonge) [inline Gemfile.lock]: these shouldnt be checked in

**Comment by eddiemonge** [comment]
add `Thumbs.db` to your global gitignore file

File: app/codelab/install-packages.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ sidebar: sidebars/codelab.html
   <img src="/assets/img/yeoman-005.png">
 </div>
 
-Let’s add some order to our list and make it sortable. For this we’re going to use Bower to install the [AngularUI Sortable module](https://github.com/angular-ui/ui-sortable), a directive component for AngularJS.
+Let’s add some order to our list and make it sortable. For this we’re going to use Bower to install the [AngularUI Sortable module](https://github.com/angular-ui/ui-sortable), a directive component for AngularJS. If you are continuing from the last step you will have to stop your site in the terminal by pressing ctrl-C.
```
Comment (eddiemonge) [inline app/codelab/install-packages.md]: `or open a new terminal window and navigate to your project folder`

**Comment by eddiemonge** [comment]
This seems good otherwise. Thanks so much!

File: Gemfile.lock (hunk 1 above)
Comment (justsayno) [inline Gemfile.lock]: I'm relatively new to git so bear with me. I have added this Thumb.db to my
global ignore and I ran "git rm cache */thumbs.db" which removed:

rm 'app/assets/img/Thumbs.db'
rm 'app/assets/img/blog/Thumbs.db'
rm 'app/assets/img/codelab/Thumbs.db'

However with the above comment on the gemfile.lock do you mean I should
just delete that line:

RedCloth (4.2.9-x86-mingw32)

From the file?

I'll fix up the wording on that other part you highlighted.

Regards,
Seth



[image: 3sparks llc] <contact@sethreid.co.nz>

Seth Reid | Web Consultant/Developer

0221705966 | contact@sethreid.co.nz

Comment (eddiemonge) [inline Gemfile.lock]: I mean this file shouldn't have the changes checked in. I don't know what adding those versions will do for everyone else including a build system.

Comment (justsayno) [inline Gemfile.lock]: Alright, I have done that in my branch. reverted the file back to the last
commit before I did work:

commit e38a18b28c0c1cc256332e796b31b5c7f7c853f6
> Author: Addy Osmani <addyosmani@gmail.com>
> Date:   Mon Apr 27 15:27:59 2015 +0100
>     Fix broken links, update stats


Have a look and see what you think. I think I have covered off all the
issues you raised.

Cheers,
Seth



[image: 3sparks llc] <contact@sethreid.co.nz>

Seth Reid | Web Consultant/Developer

0221705966 | contact@sethreid.co.nz

**Comment by eddiemonge** [comment]
want to squash your commits?

**Comment by SBoudrias** [comment]
Nice job :D Thanks a lot!

**Comment by justsayno** [comment]
I think I successfully squashed my commits. Had to learn how first :-P. Let me know if there are any issues

**Pull request merged**